### PR TITLE
config-examples: add khanelivim

### PR DESCRIPTION
**Pull request by khaneliman**
Adding personal config now that I've migrated it out of my monolithic nix configuration repo. Also included a quick typo fix

### CODE REVIEW

**Comment by GaetanLepage** [comment]
Thanks !
For the table formatting, I would keep it "compact". The rest seems fine !

**Comment by khaneliman** [comment]
Alright, pulled that formatting change.

**Review by GaetanLepage — APPROVED** [review]
LGTM, thanks !

**Review by GaetanLepage — CHANGES_REQUESTED** [review]
Wait, you need to add the actual link to your repo below.

**Comment by khaneliman** [comment]
> Wait, you need to add the actual link to your repo below.

lol sorry..

File: docs/user-guide/config-examples.md [hunk 1]
```diff
@@ -39,6 +40,7 @@ all of the nixvim options are accessible no matter how you are using it (flake,
 [GaetanLepage/nix-config]: https://github.com/GaetanLepage/nix-config/tree/master/home/modules/tui/neovim
 [gwg313/nvim-nix]: https://github.com/gwg313/nvim-nix
 [hbjydev/hvim]: https://github.com/hbjydev/hvim
+[khaneliman/khanelivim]: https://github.com/khaneliman/khanelivim
```
Comment (MattSturgeon) [inline docs/user-guide/config-examples.md:43]: Unimportant nit: could you keep this list sorted alphanumericly?

I.e. J before K

File: docs/user-guide/config-examples.md [hunk 2]
```diff
@@ -14,11 +14,12 @@ all of the nixvim options are accessible no matter how you are using it (flake,
 |-|-|
 | [ahwxorg/nixvim-config] | |
 | [alisonjenkins/neovim-nix-flake] | |
-| [dc-tec/nixvim] | NixVim conifg, inspired by some of the contributors in this list and LazyVIM |
+| [dc-tec/nixvim] | NixVim config, inspired by some of the contributors in this list and LazyVIM |
 | [elythh/nixvim] | |
 | [GaetanLepage/nix-config] | Home-manager |
 | [gwg313/nvim-nix] | |
 | [hbjydev/hvim] | |
+| [khaneliman/khanelivim] | Constantly tweaked jack of all trades development focused configuration. |
```
Comment (MattSturgeon) [inline docs/user-guide/config-examples.md:22]: Idk if we're also sorting the tables? Looks like it probably mostly is

File: docs/user-guide/config-examples.md (hunk 1 above)
Comment (khaneliman) [reply docs/user-guide/config-examples.md]: You know I can't read...

**Review by MattSturgeon — APPROVED** [review]
LGTM, thanks!

**Comment by MattSturgeon** [comment]
@mergifyio queue

**Pull request merged**